aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/AwsHttpServletResponse.java

```diff
@@ -259,13 +259,13 @@ public void setStatus(int i) {
     }
 
 
-    @Override
-    @Deprecated
-    public void setStatus(int i, String s) {
-        if (!canSetHeader()) return;
-        statusCode = i;
-        statusMessage = s;
-    }
+//    @Override
+//    @Deprecated
+//    public void setStatus(int i, String s) {
+//        if (!canSetHeader()) return;
+//        statusCode = i;
+//        statusMessage = s;
```

Comment:
looks like `statusMessage` variable can be remove completely

Reply:
Done